components/tree/index.jsx

```diff
@@ -57,7 +58,7 @@ class Tree extends React.Component {
 		// There is no need to render when blurring a node because focus is either:
 		//  - outside of the tree, or
 		//  - focused on another node in the tree, which triggers its own render
-		if (this.state.treeHasFocus && !nextState.treeHasFocus) {
+		if (!nextState.treeHasFocus) {
```

Comment:
it turns out that this is all we need, and before it was causing the issue described in https://github.com/salesforce/design-system-react/issues/1356